### PR TITLE
Introduction to ROSA book

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR adds the Introduction to ROSA book to the ROSA with HCP topic map to mirror the **[password-protected preview](https://docs.openshift.com/rosa-hcp/rosa_architecture/)**:
image

**[Preview Build](https://file.corp.redhat.com/eponvell/Upgrading-HCP/rosa_architecture/legal-notice.html)**

### CODE REVIEW

File: scripts/ocpdocs/_previewpage [hunk 1]
```diff
@@ -65,7 +65,7 @@
         </div>
         <div class="row">
           <div class="col-sm-6 text-center">
-            <h2>OpenShift ROSA</h2>
+            <h2>OpenShift ROSA (classic architecture)/h2>
```
Comment (aireilly) [inline scripts/ocpdocs/_previewpage:68]:
```suggestion
            <h2>OpenShift ROSA (classic architecture)</h2>
```

**Review by aireilly — COMMENTED** [review]
Verify the `_previewpage` file change by copying the file somewhere as `_previewpage.html` and checking it in web browser.

File: rosa_architecture/rosa_policy_service_definition/rosa-hcp-service-definition.adoc [hunk 2]
```diff
@@ -58,5 +73,18 @@ include::modules/rosa-sdpolicy-security.adoc[leveloffset=+1]
 [id="additional-resources_rosa-hcp-service-definition"]
 == Additional resources
 
-* See xref:../rosa_policy_service_definition/rosa-policy-process-security.adoc#rosa-policy-process-security[Understanding process and security for ROSA] for the latest compliance information.
-* See xref:../rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[ROSA life cycle]
+* See
```
Comment (jneczypor) [inline rosa_architecture/rosa_policy_service_definition/rosa-hcp-service-definition.adoc:76]: These two links aren't working for me on my end.

**Review by jneczypor — COMMENTED** [review]
I only found one place where links weren't working for me. Otherwise, LGTM.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**